pkg/appgw/requestroutingrules.go

```diff
@@ -455,3 +455,11 @@ func preparePathFromPathType(path string, pathType *networking.PathType) string
 	}
 	return path
 }
+
+// Application Gateway doesn't allow exact path for "/"
+// Code="ApplicationGatewayPathRulePathShouldHaveNonEmptyMatchValue" Message="Path / should have a nonempty match value followed by '/'  in PathRule xxxx
+// So, Path "/" with pathType:Exact cannot be added into the path rule. Thus, we don't support it.
```

Comment:
We might need to update the doc on it when Cx configure like it

Reply:
yes